### PR TITLE
Sharing on whatsapp

### PR DESCRIPTION
**Pull request by prdm0**
Informing that it is now possible to share via WhatsApp.

### CODE REVIEW

**Comment by cderv** [comment]
Hi. 

What is the aim of the PR ? 

I set to false so that isn't activated by default in project. 
Only if users wants to. 

The documentation reflects the default. 

Is it not ok ?

File: inst/examples/03-formats.Rmd [hunk 1]
```diff
@@ -110,7 +111,7 @@ bookdown::gitbook:
       instapaper: no
       vk: no
       whatsapp: no
```
Comment (cderv) [inline inst/examples/03-formats.Rmd:112]: See here. I added already.

**Comment by prdm0** [comment]
Hello @cderv, I just want to make it clear to people that sharing on Whatsapp with one click is possible. Several students asked how they would share on Whatsapp. Perhaps new readers, seeing this, will quickly understand that this is possible. Whatsapp is used a lot here and several other places.

Feel free to reject my pull.

Thanks.

**Comment by prdm0** [comment]
Perhaps in your R Markdown Cookbook, you could quote.

**Comment by cderv** [comment]
I already added whatapps to the documentation. So now it is there  twiced 😅

So I have nothing against your PR. It is just already there on the doc.

File: inst/examples/03-formats.Rmd (hunk 1 above)
Comment (prdm0) [reply inst/examples/03-formats.Rmd:112]: Oww, sorry, True. I need to stop programming for today. =)

Comment (cderv) [reply inst/examples/03-formats.Rmd:112]: No worries! 
Thanks for your ideas and suggestions. They are all really welcome !

**Pull request closed without merging**